cli/azd/internal/version.go

```diff
@@ -6,31 +6,47 @@ package internal
 import (
 	"regexp"
 	"strings"
+
+	"github.com/blang/semver/v4"
 )
 
-// Version is the version string printed out by the `Version` command.
-// It's updated using ldflags in CI.
+// cDevVersionString is the default version that is used when [Version] is not overridden at build time, i.e.
+// a developer building locally using `go install`.
+const cDevVersionString = "0.0.0-dev.0 (commit 0000000000000000000000000000000000000000)"
+
+// The version string, as printed by `azd version`.
+//
+// This MUST be of the form "<semver> (commit <full commit hash>)"
+//
+// The default value here is used for a version built directly by a developer when running
+// `go install`
```

Comment:
Do you mean `go build` here?

```suggestion
// `go build`
```

Reply:
It's true for either (my general workflow is to use `go install` which logically does a `go build` and then places the place it in `GOBIN` which I have added first in my path).  I'll augment the comment.